src/components/badges/badges.stories.tsx

```diff
@@ -2,6 +2,7 @@ import { Story, Meta } from '@storybook/react/types-6-0';
 import React from 'react';
 import Badge, { BadgeProps } from './badge';
 import Dot, { DotProps } from './dot';
+import Blink, { BlinkProps } from './blink/blink';
```

Comment:
let's not import from the insides of the module
we have the `index.ts`, so `./blink` could be more appropriate

Reply:
true!